src/pages/programs/program-solution/program-solution/program-solution.html

```diff
@@ -0,0 +1,20 @@
+<!-- Generated template for the ProgramSolutionComponent component -->
+<div>
+  <ion-card (click)="goToProgramSolEntity(i)">
+    <ion-item text-wrap>
+      <div class="_flex-box">
+        <div style="flex: 1;">
+          <h2 *ngIf="showProgram">{{program?.name}}</h2>
+          <p>
+            {{solution.name}}
+          </p>
+        </div>
+        <ion-icon
+          name="arrow-forward"
+          class="menuIcon"
+          *ngIf="!homePage"
```

Comment:
it will be required to hide arrow button in home component and show some other icon. 
we can remove in future if not using this previously in home we are showing different menu so i thought to keep it .